reframe/core/logging.py

```diff
@@ -419,6 +419,7 @@ def __init__(self, logger=None, check=None):
                 'check_name': 'reframe',
                 'check_jobid': '-1',
                 'check_job_completion_time': None,
+                'check_job_unix_completion_time': None,
```

Comment:
@vkarak, I used the word unix, because one could use it to send this info thru Graylog and `unix` is better than `raw`.  
What do you think about `check_job_completion_time_unix`?

Reply:
@victorusu No problem. Just put `_unix` at the end.